src/main/java/org/folio/rest/impl/OverdueFinePoliciesAPI.java

```diff
@@ -24,6 +29,8 @@ public class OverdueFinePoliciesAPI implements OverdueFinesPolicies {
   static final String DUPLICATE_ERROR_CODE = "feesfines.policy.overdue.duplicate";
   private static final String DUPLICATE_NAME_MESSAGE =
     "The Overdue fine policy name entered already exists. Please enter a different name.";
+  private static final String NEGATIVE_QUANTITY_MESSAGE =
+    "the values must be greater than or equal to 0";
```

Comment:
values of what? and why plural?

Reply:
I took the message from my ticket "Expected Results: A warning that the values must be greater than or equal to 0 appears"
Probably it should looks like "the values overdueFine and overdueRecallFine must be greater than or equal to 0 appears" ?